### ISSUE TITLE
Create NSIS Installer

### ISSUE THREAD

**Issue by rcmaehl** [rcmaehl/WhyNotWin11#546]
Create NSIS installer, including both MSI and EXE versions.

**Comment by HackerDaGreat57** [comment]
There's really no need for an installer in my opinion, because it's just a run-once app that tells you what's OK and what's not. It's not like MS Word for which an installer is critical.

At the most, the user would just pin the app to the taskbar. Simple.

**Comment by rcmaehl** [comment]
> There's really no need for an installer in my opinion, because it's just a run-once app that tells you what's OK and what's not. It's not like MS Word for which an installer is critical.
> 
> At the most, the user would just pin the app to the taskbar. Simple.

True, I'm no full sysadmin my main thought was a .msi for deployments if needed. If you have any feedback on that let me know.

**Comment by htcfreek** [comment]
A msi doesn't make much sense for deployment. In sccm and other deployment tools we (sysadmins) could simply copy and run the tool. Then we can copy the logs on a centralized place. So I don't see a need for an installer.

**Comment by rcmaehl** [comment]
With multiple people saying it's not needed, I've gone ahead and cancelled this idea.
